lazybytez/jojo-bot-website
# Switch CI package manager to Yarn

## Description
 - Switch package manager of CI workflow to yarn
 - Adjust ESLint and Prettier ignore files

## Related issue
Resolves #23 

## How can this be tested?
 - Check that CI now uses yarn in the checks of this PR
 - Ensure that `/dist` is now in the `.prettierignore` and `.eslintignore` files
